src/Microsoft.AspNetCore.Rewrite/Operands/IntegerOperand.cs

```
@@ -0,0 +1,49 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+using System;
+
+namespace Microsoft.AspNetCore.Rewrite.Operands
+{
+    public class IntegerOperand : Operand
+    {
+        public int Value { get; set; }
+        public IntegerOperationType Operation { get; set; }
+
+        public bool CheckIntegerOperand(string concatTestString)
+        {
+            int compValue;
+            if (!Int32.TryParse(concatTestString, out compValue))
+            {
+                throw new FormatException();
```

Add a message or just use Parse and let it throw